storybook/stories/API/chart/RadialBarChart.stories.tsx

```diff
@@ -25,3 +27,59 @@ export const Simple = {
     data: pageData,
   },
 };
+
+export const WithCustomizedClickLegendEvent = {
+  render: (args: Record<string, any>) => {
+    const { data } = args;
+    const [selectedRadialBar, setSelectedRadialBar] = useState('35-39');
+
+    return (
+      <ResponsiveContainer width="100%" height="100%">
+        <RadialBarChart
+          width={500}
+          height={300}
+          cx={150}
+          cy={150}
+          innerRadius={20}
+          outerRadius={140}
+          barSize={10}
+          data={data}
+        >
+          <RadialBar background dataKey="uv">
+            {data.map((entry: { name: string }) => (
+              <Cell key={`cell-${entry.name}`} opacity={selectedRadialBar === entry.name ? 1 : 0.1} />
+            ))}
+          </RadialBar>
+          <Legend
```

Comment:
I really like this example. There is only one question: Is this a story for the legend, or for the radialBar? :D 

This is good to merge, i am just wondering how we best organise storybook. Curious to hear your thoughts.

Reply:
Eh yeah it kind of applies to both but most of the customization power goes to Legend here looking at the implementation. Either way we can move it later if we want

Reply:
It would be great if we could tag stories, and users could just filter by tag, i.e. show all legend stories.

Reply:
https://github.com/storybookjs/storybook/issues/7711

Doesn't seem to have any movement but agreed